examples/getting-started/README.md

```diff
@@ -3,83 +3,162 @@
 The getting started example within this directory covers a variety of key features/components of working with CMS React within a HubSpot website. This guide outlines the specifics of using CMS React primitives such as Modules, Fields, and Islands, as well as integrating third-party dependencies, styling, assets, and setting up local development tooling for your project.
 
 ## Setup
-
 ### Environment
 
 To set up your development environment to work with CMS React, you’ll need the following:
 
 - A HubSpot account with CMS Free, Starter, Pro, or Enterprise access
 - Node.js v18.0 or higher and the associated version of NPM
 
-## Creating and Deploying a CMS React Project
+## Creating and deploying a CMS React project
+Before we dive into running your first CMS React project, its important to understand 3 significant directories that you will come across in all CMS React projects:
 
-### 1. Clone the Repo
+- `*-project`: The HubSpot project directory that gets uploaded via the Projects system to build and deploy your CMS React code found in the `/src` directory.
```

Comment:
It might be worth pointing out that these naming conventions are not required in order for the system to work, rather just a guide for organization.

Reply:
https://github.com/HubSpot/cms-react/pull/75/commits/b356a736697fd75f100c7b8779cde970535b8065